client/signup/config/steps-pure.js

```diff
@@ -808,6 +808,17 @@ export function generateSteps( {
 		},
 
 		// Woocommerce Install steps
+		'store-address': {
+			stepName: 'store-address',
+			props: {
+				headerTitle: i18n.translate( 'Add an adress so you can get paid' ),
```

Comment:
@SaxonF I understand that this is the wording as shown in the figma file. I'm curious what you think about "Add an address to accept payments" to make it a little less blunt? Or something along those lines?
cc @ianstewart

Reply:
I like that better @obenland 👍